packages/dd-trace/src/profiling/profiler.js

```diff
@@ -36,9 +36,16 @@ class Profiler extends EventEmitter {
     this._logger = config.logger
     this._enabled = true
 
+    // Log errors if the source map finder fails, but don't prevent the rest
+    // of the profiler from running without source maps.
+    let mapper
     try {
-      const mapper = await maybeSourceMap(config.sourceMap)
+      mapper = await maybeSourceMap(config.sourceMap)
```

Comment:
Should a test be added to validate that the profiler can indeed work without a source map?

Reply:
Most of the tests are already working without source maps. It's not the lack of source maps that is the issue it's that if you run it with certain system process managers it might be run with / as the working directory and end up trying to scan parts of the filesystem it should not. Not sure how we'd write a test for that. 🤔

Reply:
The idea of the test would be to make sure profiling still works for that type of use case instead of being completely disabled.